webpack/assets/javascripts/react_app/components/common/table/index.js

```diff
@@ -0,0 +1,43 @@
+import { Table as PfTable } from 'patternfly-react';
```

Comment:
Could you use the refactored table from katello?

Reply:
What do you mean by that :smile: ? This is the `<Table>` component that I took from `Katello`.

Reply:
We refactor the table lately :disappointed: 

https://github.com/Katello/katello/pull/7467